crates/apis/src/stream_io/io.js

```diff
@@ -19,7 +19,7 @@
       }
       return __javy_io_writeSync(
         fd,
-        data.buffer,
+        data,
```

Comment:
Changed this to make it easier to work with the JS API from the Rust side. In theory we could change the entire API so that Rust only receives `data`, given that we're certain at this point that this is an `TypedArray` and therefore we can derive the `byteOffset` and `byteLength`. This can be done in a follow-up though.